Elections_scraper/election_sc.py

```diff
@@ -0,0 +1,158 @@
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=552542&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=598101&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=511951&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=552607&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=598135&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=598003&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=598143&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=598160&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=512192&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=511986&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=552631&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=512176&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=598232&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=598691&xvyber=8102',
+        'https://volby.cz/pls/ps2017nss/ps311?xjazyk=CZ&xkraj=14&xobec=598259&xvyber=8102',
+        ]
+
+def second():
+    infos = []
+    elections = []
+    for url in URLS:
+        answer = get_answer_2(url)
+        parsed = pull_data_2(answer)
+        header_div = parsed.find_all('table', id='ps311_t1', class_='table')
+        election_div = parsed.find_all('div', class_='t2_470')
+        rows = find_row_2(election_div)
+        election = [election_info(row) for row in rows]
+        info = county_info(header_div)
+        infos.append(info)
+        elections.append(election)
+        save_csv_2(infos,elections)
+
+
+def get_answer_2(url):
+    return requests.get(url)
+
+def pull_data_2(ans):
+    return bs(ans.text, "html.parser")
+
+def find_row_2(tables):
+    return [
+        row
+        for table in tables
+        for row in table.find_all('tr')[2:]
+    ]
+
+def county_info(parsed):
+    for contain in parsed:
+        people = contain.find('td',class_='cislo', headers='sa2').text
+        is_env = contain.find('td',class_='cislo', headers='sa3').text
+        submitted = contain.find('td',class_='cislo', headers='sa5').text
+        return {
+            "Volici v seznamu":people,
+            "Vydané obálky":is_env,
+            "Platné hlasy":submitted
+        }
+def election_info(tr) -> dict:
+    try:
+        party = tr.find_all("td")[1].text
+        votes = tr.find_all('td')[2].text
+        return {
+            "PARTY": party,
+            "VOTES": votes
+        }
+    except AttributeError:
+        print(f'Wrong assigned indexes')
+
+def save_csv_2(data, data2):
```

Comment:
Can you think of a better name? Like `save_election_data` or something? The same for `data` and `data2`. It's hard to guess what should be there.